analyzers/backend/docker/container.go

```
@@ -62,10 +64,17 @@ func (d *DockerClient) StartDockerContainer() error {
 		},
 	}
 
+	// Check if platform is empty or not
+	imageArch := d.ImagePlatform
+	if d.ImagePlatform == "" {
+		log.Println("received no image platform. using current OS architecture as image's architecture.")
```

Should we log this; or silently fallback to use the OS architecture?

---

Missed this. In my opinion, we should not log this.

---

Resolved in https://github.com/deepsourcelabs/cli/pull/156/commits/e7fadee42f01bd42dfc57499d5b623c38c1274bb.

---

Thanks! Can you please rebase with `analyzers` branch?

---

Sure!